waku.nimble

```diff
@@ -65,7 +65,7 @@ task test1, "Build & run Waku v1 tests":
 
 ### Waku v2 tasks
 task wakunode2, "Build Waku v2 (experimental) cli node":
-  buildBinary "wakunode2", "waku/v2/node/", "-d:chronicles_log_level=DEBUG"
+  buildBinary "wakunode2", "waku/v2/node/", "-d:chronicles_log_level=TRACE"
```

Comment:
This is fine for local checkout but for merging can remove this

Reply:
done